src/containers/SetsContainer/index.js

```diff
@@ -36,9 +38,21 @@ const SetsContainer = () => {
   const handleToggleSideSheet = () => setSideSheetOpen(!sideSheetOpen);
   const handleToggleSearch = () => setSearchOpen(!searchOpen);
 
+  const { search } = useLocation();
+  const parsedSearch = queryString.parse(search);
+
+  const ougIdsFromParams = parsedSearch.orgUnitGroups.split(",").filter(i => i);
```

Comment:
Needed to add the `(parsedSearch.orgUnitGroups || "").split(",").filter(i => i);` here as well